capa/render/proto/capa.proto

```diff
@@ -327,7 +313,8 @@ message Sample {
 }
 
 enum Scope {
-  SCOPE_UNSPECIFIED = 0;
+  SCOPE_UNSPECIFIED = 0;  // TODO do differently so json conversion works, currently gives `"scope": "SCOPE_FUNCTION"` which pydantic cannot parse
+                          // could just make string?! and assert in code that it's one of supported values?!
```

Comment:
agree that this works well in python/json, but using an enum would be much easier on the c/c++ side. i don't know what we should be optimizing for, yet.

Reply:
We can use enums. This comment came about when comparing the proto-converted json to the rd-json (with the hope that we can easily compare them). I now think we'll need a custom converter anyway (at least for testing both formats are equal).